### ISSUE TITLE
Aremi logo missing alt text

### ISSUE THREAD

**Issue by chloeleichen** [NICTA/aremi-natmap#20]
The Aremi logos are missing alt text, which is a WCAG 2.0 AA requirement. 

![accessibility](https://cloud.githubusercontent.com/assets/7508939/10358882/c1abf226-6dde-11e5-8f6e-5532ac52250e.png)
